py/selenium/webdriver/remote/webdriver.py

```diff
@@ -1211,7 +1212,9 @@ def start_devtools(self):
             return self._devtools, self._websocket_connection
         if self.caps["browserName"].lower() == "firefox":
             raise RuntimeError("CDP support for Firefox has been removed. Please switch to WebDriver BiDi.")
-        self._websocket_connection = WebSocketConnection(ws_url)
+        self._websocket_connection = WebSocketConnection(
+            ws_url, self.client_config.websocket_timeout, self.client_config.websocket_interval
```

Comment:
```suggestion
            ws_url,
            self.command_executor.client_config.websocket_timeout,
            self.command_executor.client_config.websocket_interval,
```

All BiDi tests are failing because of this.

Reply:
Thanks, I'll take a look and fix it... I didn't see anywhere else in the code this was getting called, but I must have missed something.

Reply:
fixed